src/mixins/Helper.js

```diff
@@ -7,14 +7,6 @@ import * as consts from '@/consts/consts'
 export default {
```

Comment:
Not sure that this helper file is very helpful. Probably needs to go the same way as alerts!

Reply:
Agreed, almost went ahead and killed it completely, more than happy to do so.

Reply:
Definitely. Kill it off.